app/femr/ui/models/triage/IndexViewModelPost.java

```diff
@@ -77,14 +78,20 @@ public void setFirstName(String firstName) {
         this.firstName = firstName;
     }
 
-    public String getLastName() {
-        return lastName;
-    }
+    public String getLastName() { return lastName; }
```

Comment:
Did you mean to commit this style change? It makes it inconsistent with the others and seems unrelated to the task.